debian/cocaine-v12-runtime.init

```diff
@@ -14,7 +14,7 @@
 # Author: Andrey Sibiryov <kobolog@yandex-team.ru>
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
-NAME=cocaine-runtime
+NAME=cocaine-v12-runtime
```

Comment:
It's a long story about conductor, repos and all of that. May be you remember the conductor? It can't handle a ticket (even create) if a package is not in all repos. So we had cocaine-runtime < 12 in common-precise repo and cocaine-runtime > 12 in our private repo cocaine-v12-precise. It makes impossible to use conductor to install the package. 
Long story short it's a Yandex related problem.